pkg/crc/api/api.go

```diff
@@ -69,12 +69,34 @@ func (api CrcApiServer) handleConnection(conn net.Conn) {
 		logging.Error("Error decoding request: ", err.Error())
 		return
 	}
-
-	if handler, ok := api.handlers[req.Command]; ok {
-		result := handler(req.Args)
-		writeStringToSocket(conn, result)
+	if req.Command == "start" || req.Command == "stop" || req.Command == "delete" {
+		// queue new request to channel
+		if !addRequestToChannel(req, requests) {
+			defer conn.Close()
+			logging.Error("Channel capacity reaced, unable to add new request")
+			writeStringToSocket(conn, "Channel capacity reaced, unable to add new request")
```

Comment:
"reached" twice

Reply:
Ahh, this i fixed in the last commit a9056c9bea085dcf490b7915a505b646a7703fb7

Reply:
done..